databases/week3/hyf_lesson3_mealsharing.sql

```diff
@@ -0,0 +1,223 @@
+CREATE DATABASE hyf_lesson3_mealsharing
+DEFAULT CHARACTER SET = 'utf8mb4';
+USE hyf_lesson3_mealsharing;
+CREATE TABLE `meal` (
+  `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` VARCHAR(255) NOT NULL,
+  `description` TEXT NOT NULL,
+  `location` VARCHAR(255) NULL,
+  `time` DATETIME NOT NULL,
+  `max_reservations` INT(10) NOT NULL,
+  `price` DECIMAL(7, 2) NOT NULL,
+  `created_date` DATE NOT NULL
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `reservation` (
+  `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `number_of_guests` INT(10) NOT NULL,
+  `created_date` DATE NOT NULL,
+  `contact_phonenumber` VARCHAR(255) NOT NULL,
+  `contact_name` VARCHAR(255) NOT NULL,
+  `contact_email` VARCHAR(255) NOT NULL,
+  `meal_id` INT(10) UNSIGNED NOT NULL,
+  CONSTRAINT `fk_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
If we delete all reservations for a given meal, the meal entry will also be deleted, is that what you want? (It is ok, just checking )

Reply:
i fix the problem